src/commands/bans/ban.ts

```diff
@@ -0,0 +1,31 @@
+import {Command} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+import {ban as banImpl} from "../../controllers/BanController";
+import {ConfigKey, getConfig} from "../../util/Config";
+
+const ban: Command = {
+    name: "ban",
+    description: "Bans a word",
+    usage: "ban <word>",
+    procedure: async (client: Client, message: Message, args: string[]) => {
```

Comment:
Not sure which version of TS/ECMA you're on, but is top-level `await` permitted now?

Reply:
james. that says `async`.

Reply:
Did I stutter?